website/docs/sys-mgmt/background-tasks.md

```diff
@@ -0,0 +1,78 @@
+- **Running**: the task is currently running.
+
+## Manage background tasks
+
+### View system tasks
+
+You can view and manage all background tasks and schedules from the Admin interface.
+
+However, by default, tasks are shown _as close as possible_ to their relevant objects. For instance, LDAP source synchronization tasks and schedules are shown on the LDAP source detail page.
+
+When a task or a schedule cannot be associated to an object (for example, housekeeping tasks), it is referred to as "standalone" and is displayed under **Dashboards** > **System Tasks**. Note that tasks created from a schedule are associated to that schedule and thus are not considered standalone. Both schedule and task items can be expanded to view additional details about them.
+
+If you cannot find the object to which a task or schedule is attached, deselect the "Show only standalone tasks/schedules" toggle on the **System Tasks** page to show all tasks and schedules, including the ones that are attached to objects.
+
+By default, successful tasks are hidden to minimize the number of shown items. Deselect "Exclude successful tasks" to display them.
+
+### Schedule configuration
+
+When the authentik system creates a schedule it is assigned a default interval. The schedule uses a format based on [unix-cron](https://man7.org/linux/man-pages/man5/crontab.5.html).
+
+To change that interval, click the Edit icon for the specific schedule and update it.
+
+:::warning
+Some tasks are required to run at regular intervals. For tuning reasons we recommend editing the intervals only for synchronization schedules, not for other types of schedules.
+:::
+
+Schedules can also be _paused_ to prevent new tasks to be created from them. They can still be triggered manually while paused. When you un-pause a schedule, it will be triggered immediately.
+
+### Failed tasks
+
+When a task fails, i.e. when the code throws an exception, the task will be retried up to the value configured in [`AUTHENTIK_WORKER__TASK_MAX_RETRIES`](../install-config/configuration/configuration.mdx#authentik_worker__task_max_retries). Tasks that self-reported an error or a warning will not be retried.
+
+Failed tasks will be displayed like any other tasks. Each task can be expanded to show its logs. The logs are split into two parts: "Current execution logs" for the current execution, and "Previous execution logs" for logs from previous executions that happened before a retry was initiated. The information contained in the logs indicate either a transient error (a network connection failed for example), a mis-configuration (wrong password set in the LDAP source for example), or a bug in authentik.
+
+#### Restarting tasks
+
+To restart a task, click the retry arrow next to the task. It will be queued again and picked up by a worker.
+
+:::info
+To retry tasks created from a schedule, we recommend manually triggering the schedule instead of restarting one of its tasks.
```

Comment:
```suggestion
To retry tasks created from a schedule, we recommend manually triggering the schedule (click the Run arrow beside the schedule) instead of restarting one of its tasks.
```